### PR TITLE
tensorRT support for BatchNormalization 

### PR DESCRIPTION
**Pull request by lgarithm**
### Checklist
- [ ] I've tested that my changes are compatible with the latest version of Tensorflow.
- [ ] I've read the [Contribution Guidelines](https://github.com/tensorlayer/tensorlayer/blob/master/CONTRIBUTING.md)
- [ ] I've updated the documentation if necessary.

### CODE REVIEW

File: tensorlayer/layers/normalization.py [hunk 1]
```diff
@@ -67,6 +69,52 @@ def __init__(
...
+    if data_format == 'NHWC':
+        return x * b
+    elif data_format == 'NCHW':
+        return x * _to_channel_first_bias(b)
+    else:
+        raise ValueError('invalid data_format: %s' % data_format)
+
+
+def _bias_add(x, b, data_format):
+    """Alternative inplementation of tf.nn.bias_add which is compatiable with tensorRT."""
```
Comment (luomai) [inline tensorlayer/layers/normalization.py]: inplementation -> implementation

**Comment by DEKHTIARJonathan** [comment]
Why removing bias_add? It doesn't seem to be incompatible with TENSORRT?

https://docs.nvidia.com/deeplearning/sdk/tensorrt-api/python_api/workflows/tf_to_tensorrt.html

**Comment by DEKHTIARJonathan** [comment]
Wait before merging please. We are changing all layers for TL 2.0...
I prefer finishing this PR first. It's already complex, we don't need an increase in complexity

**Comment by luomai** [comment]
@DEKHTIARJonathan We are adding minimal but necessary features that on-going TL-related projects are required, and also bug fix. They are mostly regarding to the distributed training and the openpose-plus project. The openpose-plus project is to make TL fully completely compatible with TensorRT 4.0.

**Comment by luomai** [comment]
@DEKHTIARJonathan We will merge this PR into your model-api PR. No worry. It shall be simple.

**Comment by DEKHTIARJonathan** [comment]
We have a problem. We use tf.nn.bias_add everywhere. If this is problem, this problem is everywhere

**Comment by lgarithm** [comment]
@DEKHTIARJonathan 
This bug will only be trigged when you are converting the model to tensorRT and `b` is not a constant in `tf.nn.bias_add`. The fix here will be just enough for current use case.

And this is just a temporary solution. The long term solution would be fixing `convert_bias_add` in tensorRT. It should be easy but the code is not open sourced.

**Comment by luomai** [comment]
Could you update the changelog? @lgarithm
